test/transforms.jl

```diff
@@ -10,6 +10,12 @@
     f = rand(4000)
     t = Table(; a, b, c, d, e, f)
 
+    x1 = rand(4000)
+    x2 = rand(4000)
+    y1 = rand(4000)
+    y2 = rand(4000)
+    t₂ = Table(; x1, x2, y1, y2)
+
```

Comment:
Let's move this to right before the corresponding test lines. We should probably stick to the surrounding convention and use t for the variable name. Every test set overwrites a new t and runs a couple of tests.

Reply:
OK, done!